### ISSUE TITLE
NCD report email epi weeks to months

### ISSUE THREAD

**Issue by jsoppela** [meerkat-code/meerkat_frontend#143]
In the email title: Epi week 39 -> Monthly Report
In the email text: Epidemiological Week 39 -> the previous month

**Comment by jsoppela** [comment]
@jonathansberry has already done the implementation

**Comment by jsoppela** [comment]
This issue is still relevant in production

**Comment by jsoppela** [comment]
It seems the issue is with the email subject. The current code for designating the subject will display epi week as it should be epi month in the case of NCD report.

**Comment by jsoppela** [comment]
The fixes are committed in api repo branch #issue143 but a series of test emails still needs to be sent

**Comment by jsoppela** [comment]
The email subject when currently sent is:
Jordan | Non Communicable Disease**s** Report (01 November 2016 - 30 November 2016)

The requirement is to change it to:
Jordan | Non Communicable Disease **Monthly** Report (01 November 2016 - 30 November 2016)

The email subject is built as 
```
subject = '{country} | {title} ({start_date} - {end_date})'.format(
                country = gettext(country),
                title = gettext(report_list[report]['title']),
                start_date = format_datetime(start_date, 'dd MMMM YYYY'),
                end_date = format_datetime(end_date, 'dd MMMM YYYY')
            )
```
I'll change this to 
```
subject = '{country} | {email_title} ({start_date} - {end_date})'.format(
                country = gettext(country),
                title = gettext(report_list[report]['email_title']),
                start_date = format_datetime(start_date, 'dd MMMM YYYY'),
                end_date = format_datetime(end_date, 'dd MMMM YYYY')
            )
```
, add a branch issue143 to meerkat_jordan with an additional report configuration parameter "email_title"

**Comment by jsoppela** [comment]
The branch has been merged to master and the code is ready to be deployed. This issue can be closed once the deployment is done.

**Comment by jsoppela** [comment]
The code has been deployed
